frontend/src/lib/search-utils.test.ts

```diff
@@ -66,4 +82,34 @@ describe('search-utils sermver operator', () => {
   it('can coerce incomplete semver strings', () => {
     expect(handleSemverOperatorComparison(versionIncomplete, versionOne, SearchOperator.Equals)).toBeTruthy()
   })
+  it('can infer on incomplete semver equals', () => {
+    expect(handleSemverOperatorComparison(versionOne, versionOneIncomplete, SearchOperator.Equals)).toBeTruthy()
+  })
+  it('can infer on incomplete semver greater than', () => {
```

Comment:
This test fails with my proposed PR, which makes sense if we consider >1 to mean 2 or higher. (1.5.0 is the first arg here)

By the way, using constants for the test values here just makes it hard to understand and debug the tests. I would just repeat the numbers in the tests.